statics/js/components/topology.js

```diff
@@ -299,6 +299,10 @@ var TopologyComponent = {
       });
     }
 
+    if (typeof(this.$route.query.topology_legend_hide) !== "undefined") {
```

Comment:
I would prefer to truncated to gremlin expression, adding a title attribute to have tooltip with the full expression

Reply:
its' actually required in our scenarios in which we embed skydive into other views as an iframe that one can eliminate various elements (such as the legend), so I have left the commit, but I also added something similar to what you are suggesting for better handling of text overflow. I considered three alternatives:
(1) tooltip --> issue here is that this is not what tooltips typically are used for, they provide assistance not another view of the content ..
(2) auto scrolling on mouse hover --> while initially appearing agent the animation makes it difficult to pause a read a specific area of the Gremlin filter ... and also copy of the content to the clipboard is not (easily) possible.
(3) manual scrolling --> this works well, the scrollbar was eliminate to real estate